mir_planning/mir_actions/mir_perceive_location/ros/scripts/perceive_location_server.py

```diff
@@ -199,6 +200,31 @@ def execute(self, userdata):
 
 # ===============================================================================
 
+class SetPerceptionParams(smach.State):
+    def __init__(self):
+        smach.State.__init__(
+            self,
+            outcomes=["success", "failure"],
+            input_keys=["goal"],
+        )
+        self.set_named_config = gbs.set_named_config("multimodal_object_recognition_atwork")
+
+    def execute(self, userdata):
+        obj_category = Utils.get_value_of(userdata.goal.parameters, "obj_category")
+
+        rospy.loginfo("=============[PERCEIVE_LOCATION] obj_category: %s", obj_category)
```

Comment:
are these loginfos needed? If yes, format them similar to others (e.g. remove "[IFFFF]")